src/pages/odd.md

```diff
@@ -9,63 +9,13 @@ headerTextColor:
+In the work with did in our [We Count project ](https://wecount.inclusivedesign.ca/)and [Future of Work and Disability project](https://wecount.inclusivedesign.ca/views/fwd/) we noticed that AI hiring tools were filtering out highly qualified candidates with disabilities. It is estimated that over 50% of companies will deploy some form of AI assisted or AI automated hiring tools in the next decade (Sage, Pego). We knew that AI hiring tools exploit data regarding past hiring to filter applicants by optimizing the characteristics of past successes. In optimizing past successes these tools are biased against diversification; while the companies deploying them have a commitment to diversity, equity, and inclusion. AI hiring tool options such as specifying “culture fit,” persona matching, team profiles, and favourite university, contribute toward creating a company monoculture. This is not good for company survival. It also filters out applicants with disabilities. Promising research has shown that algorithms that favour data exploration rather than data exploitation result in greater hiring diversity without compromising hiring success [](<(https://www.nber.org/papers/w27736>)([Hiring as Exploration](https://www.nber.org/papers/w27736)). This prior research focuses on race, origin, and gender diversity. ODD will extend this research to the more difficult area of disability. Unlike race, origin, and gender; disability has no common data characteristic other than distance from the average so that things made for the mainstream don’t work for you.
```

Comment:
You probably lost the fix for the Hiring as Exploration link, but we can fix that later.